src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -207,6 +220,41 @@ private ProcessResult createNewCase(
         }
     }
 
+    private void handlePayments(CaseDetails exceptionRecord, long newCaseId) {
+
+        boolean containsPayments =
+            Objects.equals(
+                exceptionRecord.getData().get(ExceptionRecordFields.CONTAINS_PAYMENTS).toString(),
```

Comment:
Can we extract `exceptionRecord.getData()` into a variable ? It’s repeated thrice